deploy/kubernetes/base/node.yaml

```diff
@@ -71,9 +71,12 @@ spec:
           hostPath:
             path: /var/lib/kubelet
             type: Directory
-        - name: plugin-dir
+        - name: csi-dir
           hostPath:
-            path: /var/lib/kubelet/plugins/pd.csi.storage.gke.io/
+          # Specifically not placed in deprecated plugin directory
+          # /var/lib/kubelet/plugin/ to avoid Kubelet plugin watcher mistaking
+          # the CSI Socket as a Registration Socket
+            path: /var/lib/kubelet/plugins_pd.csi.storage.gke.io/
```

Comment:
Unsure if we want to create another top level directory under "/var/lib/kubelet".  We should be able to remove plugin watcher looking at the deprecated plugin directory in 1.18. Maybe we can just wait until then?

Reply:
ack. removing in k8s so we can continue to use this directory.